blocks/author-profile/author-profile.css

```diff
@@ -0,0 +1,70 @@
+.author-profile {
+  padding-top: var(--udexSpacer20);
```

Comment:
I think it is better to use margin here - so that if the upper element uses margin as well it can collapse to the higher value

Reply:
Done.